concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1620,17 +1620,21 @@ jobs:
 
                   cp ./paas-cf/config/service-brokers/compose/catalog.json ./paas-compose-broker/
                   cd paas-compose-broker/
+
                   ruby -ryaml -e "
-                    manifest = YAML.load_file('manifest.yml')
-                    env = { 'env' => {
+                    env = {
                       'LOG_LEVEL' => 'DEBUG',
                       'USERNAME' => 'compose-broker',
                       'PASSWORD' => '${COMPOSE_BROKER_PASS}',
                       'COMPOSE_API_KEY' => '${COMPOSE_API_KEY}',
                       'DB_PREFIX' => '${DB_PREFIX}',
                       'CLUSTER_NAME' => '${CLUSTER_NAME}',
-                    }}
-                    manifest['applications'].each { |app| app.merge!(env) }
+                    }
+                    manifest = YAML.load_file('manifest.yml')
+                    manifest['applications'].each { |app|
+                      app['env'] ||= app['env']
```

Comment:
I'm puzzled as to what this line is doing. I suspect you wanted `app['env'] ||= {}`.

Reply:
ping @samcrang ... maybe something got lost in translation?

But re this ruby pattern (that is repeated a few times) generally: maybe it would be nicer to have a `make manifest.yml` task in paas-compose-broker that builds the manifest based on env, and remove the yaml/ruby noise?

Reply:
@chrisfarms Yeah @alext is right—it should be `app['env'] ||= {}`.

Reply:
Cool ... I'll make a mini PR to fix.

It won't break right now, but if someone removes the `env` section from `manifest.yaml` it will explode.

Reply:
@chrisfarms or @samcrang do you want to raise a new PR and I'll merge?